spec/lib/fetch_book_data_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+require 'vcr_helper'
+
+describe FetchBookData, type: :lib, vcr: VCR_OPTS do
+  describe '#subjects' do
+    subject { described_class.new.subjects }
```

Comment:
suggestion:  consider using named subjects 

http://www.betterspecs.org/#subject

It's not mandatory, but why not take advantage of rspec (or ruby, or any dsl) to add even more self-documentation to code one writes, esp if it's free, which it is here.